src/transformers/jscodeshift.js

```diff
@@ -0,0 +1,37 @@
+import compileModule from './utils/compileModule';
+
+var fs = require('fs');
+
+const ID = 'jscodeshift';
+
+const defaultTransform =
+  fs.readFileSync(__dirname + '/transformJscodeshift.txt', 'utf8');
+
+function transform({transformCode, code}) {
+  return new Promise((resolve, reject) => {
+    loadjs(['babel-core', 'jscodeshift'], (babel, jscodeshift) => {
+      try {
+        // This might throw
+        var transform = compileModule(
+          babel.transform(transformCode).code
```

Comment:
I felt really tempted to self-host the plugin with a previous version of the transformCode, like

```
babel.transform(transformCode, {plugins: [previousTransformVersion]});
```

:lollipop:

@sebmck